spec/factories/generic_files.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
```

Comment:
Do you want to rename the file to file_sets.rb to keep it in sync with the factory name?